cloudant-client/src/test/java/com/cloudant/tests/ViewsTest.java

```diff
@@ -1246,9 +1247,6 @@ public void staleParameterValues() throws Exception {
         UnpaginatedRequestBuilder<String, Integer> viewBuilder = database.getViewRequestBuilder
                 ("testDDoc", "testView").newRequest(Key.Type.STRING, Integer.class);
 
-        MockResponse mockResponse = new MockResponse().setResponseCode(200).setBody
```

Comment:
Why have these mock responses been removed?

Reply:
The same content was being enqueued in the `assertStaleParameter` method. So they ended being queued twice. This left extra requests in the mockwebserver that caused the new mock test to fail.

Reply:
OK, understood